### ISSUE TITLE
Flickr Fixr not working on MS Edge

### ISSUE THREAD

**Issue by StigNygaard** [StigNygaard/Stigs_Flickr_Fixr#1]
Userscript not working on MS Edge with Tampermonkey. Nothing at all happens.
I don't use Edge myself, but I like the work MS is doing with this and want to support it. I need to find out what's going wrong and a way to fix it...

**Comment by StigNygaard** [comment]
Just did some more investigation on this, and it seems even the simplest userscript wont work on Flickr using Edge/Tampermonkey. So I have concluded that either Edge or Tampermonkey has some restriction or bug that prevents userscripts from running on flickr.com.

BUT then I tried "sideloading" the new browser extension currently available for Firefox and Chrome. And guess what, it works well in MS Edge too ! Unfortunately distributing a browser extension for MS Edge in Microsoft Store seems to be a lot more trouble than it is to distribute browser extensions for Firefox and Chrome. Not even sure Microsoft currently approves anything from a private hobby project. Still, I could try. But would anyone care?
